Extensions/Testing/LoadableExtensionTemplate/LoadableModuleTemplate/CMakeLists.txt

```diff
@@ -48,7 +47,6 @@ set(MODULE_RESOURCES
 #-----------------------------------------------------------------------------
 slicerMacroBuildLoadableModule(
   NAME ${MODULE_NAME}
-  TITLE ${MODULE_TITLE}
```

Comment:
It's definitely an improvement to specify the module title in the C++ code with tr, and also to clean up the template and usage in the core.  From what I can tell older loadable modules that use the old style will still work but won't be translatable.  I wonder if we should add a warning message in `CMake/SlicerMacroBuildLoadableModule.cmake` if `TITLE` is passed, with a description of how to update the code to make it localizable.

Reply:
Good point, we should deprecate this option at some point. Now I'll just add a note to the translator's guide.

Reply:
I've added a note here: https://github.com/Slicer/SlicerLanguagePacks/blob/main/DevelopersManual.md#translating-module-title